alpine-rails-nginx/nginx.patch

```diff
@@ -0,0 +1,34 @@
+--- auto/lib/openssl/conf
```

Comment:
I'm really curious why nginx authors read openssl from `.openssl` by default. No sane persion would hide source in a hidden directory?